adapter/operations.go

```diff
@@ -83,3 +90,8 @@ func (h *Adapter) ListOperations() (Operations, error) {
 func (h *Adapter) ApplyOperation(context.Context, OperationRequest) error {
```

Comment:
And here. Due to we not use the interface, so we should comment out which methods should be implemented by each adapter.